src/index.js

```diff
@@ -349,7 +362,25 @@ class MiniCssExtractPlugin {
                     'reject(err);',
                   ]),
                   '};',
-                  'linkTag.href = fullhref;',
+                  '} else {',
+                  'var errorTimeout = 60000;',
+                  'var startTime = Date.now();',
+                  'var interval = setInterval(function(){',
+                  'if(linkTag.style){',
+                  'clearInterval(interval);',
+                  'resolve();',
+                  '}',
+                  'if(!linkTag.style && (Date.now() - startTime) > errorTimeout){',
```

Comment:
The support for onload and onerror differ. Make sure to handle them separately. Use a setTimeout for error handling and don't merge it into the setInterval for onload.